src/server/project.ts

```diff
@@ -1215,6 +1215,7 @@ namespace ts.server {
 
                     // Provide global: true so plugins can detect why they can't find their config
                     this.projectService.logger.info(`Loading global plugin ${globalPluginName}`);
+                    // tslint:disable-next-line no-object-literal-type-assertion (TODO: GH#18217)

```

Comment:
looks like we should change the interface definition  here instead.

Reply:
But the fact that the interface doesn't declare it would imply that we're not using the property?